packages/components/atoms/f-link/src/components/Link.vue

```diff
@@ -1,45 +1,127 @@
 <template>
-    <div
-        :class="$style['o-link']"
-        data-test-id="link"
-    >
-        {{ copy.text }}
-    </div>
+    <a
+        :class="[
+            $style['o-link'],
+            { [$style['o-link--bold']]: isBold },
+            { [$style['o-link--noDecoration']]: !hasTextDecoration },
+            { [$style['o-link--full']]: isFullWidth }
+        ]"
```

Comment:
You can add the conditional classes in a single object

```suggestion
            $style['o-link'], {
                [$style['o-link--bold']]: isBold,
                [$style['o-link--noDecoration']]: !hasTextDecoration,
                [$style['o-link--full']]: isFullWidth
            }]"
```